_docs/_latest/setup/upgrade.md

```diff
@@ -401,6 +401,65 @@ Forseti is updated to be usable on a non organization resource.
 {% endcapture %}
 {% include site/zippy/item.html title="Upgrading 2.3.0 to 2.4.0" content=upgrading_2_3_0_to_2_4_0 uid=5 %}
 
+{% capture upgrading_2_4_0_to_2_5_0 %}
+
+Starting v2.5, Forseti Inventory will be integrated with the Cloud 
+Asset Inventory (CAI) service if the user is deploying Forseti. CAI
+feature is supported only if the level is `organization`.
+
+Users upgrading to v2.5 from v2.4 can enable CAI by following the steps below:
+1. Enable `Cloud Asset API` under APIs & Services.
```

Comment:
If you are using the gcloud shell command below, you can also use a gcloud command to enable the api.

Reply:
True. Added gcloud command as well.